From 87b8f14811abe5b707ebe109296bcb0efbebcce2 Mon Sep 17 00:00:00 2001
From: Szabolcs Nagy <nsz@port70.net>
Date: Sat, 19 Dec 2020 22:12:50 +0000
Subject: [PATCH] signal.h: add MTE specific SIGSEGV codes from linux v5.10

add synchronouse and asynchronous tag check failure codes, see

  linux commit 74f1082487feb90bbf880af14beb8e29c3030c9f
  arm64: mte: Add specific SIGSEGV codes
---
 include/signal.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/signal.h b/include/signal.h
index 9ed929e4..f270a594 100644
--- a/include/signal.h
+++ b/include/signal.h
@@ -75,6 +75,8 @@ typedef struct sigaltstack stack_t;
 #define SEGV_ACCERR 2
 #define SEGV_BNDERR 3
 #define SEGV_PKUERR 4
+#define SEGV_MTEAERR 8
+#define SEGV_MTESERR 9
 
 #define BUS_ADRALN 1
 #define BUS_ADRERR 2
